server/src/auth0/auth0Authentication.js

```diff
@@ -1,140 +1,136 @@
-const fetch = require('isomorphic-fetch');
```

Comment:
I wonder why git thinks this whole file has changed. I hope it isn't line-ending changes. Good thing for being able to filter out whitespace changes with `w=1` in the url.